docs/reference/transform.md

```diff
@@ -1,18 +1,14 @@
 ## Transform
 
-Derived from [Group](group.md).
+Derived from [Pose](pose.md).
 
 ```
 Transform {
-  SFVec3f    translation     0 0 0     # any vector
-  SFRotation rotation        0 0 1 0   # unit axis, (-inf, inf) angle
   SFVec3f    scale           1 1 1     # any vector
-  SFFloat    translationStep 0.01
-  SFFloat    rotationStep    0.261799387
 }
 ```
 
-Direct derived nodes: [Solid](solid.md).
+Direct derived nodes: none.
 
 ### Description
```

Comment:
The description contains both the Pose specific field and the inherited ones whereas it should only contain the specific one (scale)

Reply:
But that's the case. Are you sure your are up-to-date with this branch?

Reply:
Sorry not the description but the field summary